gravitee-apim-console-webui/src/management/management.module.ajs.ts

```diff
@@ -547,19 +546,15 @@ require('diff/dist/diff.min.js');
 require('angular-loading-bar');
 
 // Highcharts
-import * as Highcharts from 'highcharts';
-(<any>window).Highcharts = Highcharts;
-import HighchartsMore from 'highcharts/highcharts-more';
-import SolidGauge from 'highcharts/modules/solid-gauge';
-import NoDataToDisplay from 'highcharts/modules/no-data-to-display';
-import Map from 'highcharts/modules/map';
-
-HighchartsMore(Highcharts);
-SolidGauge(Highcharts);
-NoDataToDisplay(Highcharts);
-Map(Highcharts);
-
-import '@highcharts/map-collection/custom/world';
+const Highcharts = require('highcharts/highstock');
+window.Highcharts = Highcharts;
+require('highcharts/modules/exporting')(Highcharts);
```

Comment:
Needed?

Reply:
It seems to me that it is. But FF was very complicated to make work in es6 🪦 . As it seems to work, I don't really want to modify it 🙈